web/src/stream_create_subscribers.js

```diff
@@ -83,28 +84,30 @@ export function build_widgets() {
 
     all_users_list_widget = ListWidget.create($("#create_stream_subscribers"), [current_user_id], {
         name: "new_stream_add_users",
+        get_item: people.get_by_user_id,
```

Comment:
Why we need this? Fetching the object in `modifier` is fine. If we want to do such refactoring that should be done in separate commits with proper reason of doing so in the commit message. This commit should only make changes needed to allow sorting of the list.

Reply:
To allow sorting we have two options:
1. Pass user_id and pass a function to get user from user_id.
2. Pass complete user object.

Without any refactoring parameters passed to sorting function will be user_ids while it is expected to be user object.

Reply:
Ahh, fine! Sounds good. I misunderstood the modifier to be handing conversion to user objects from ID.